README.md

```diff
@@ -90,3 +95,20 @@ In a separate terminal tab, run the TailwindCSS watcher for real-time style comp
 ```bash
 rails tailwindcss:watch
 ```
+
+To test Discord interactions, you will need to expose your local server using a tool like Ngrok. If you're using Ngrok, run this command (assuming your app runs on port 3000):
+```bash
+ngrok http 3000
+```
+
+Then, paste the HTTPS link into the application's Discord dashboard. You should see two requests pointing to `/api/discord/interactions`: one with a `200` status code response and another with a `401`
+<img width="979" alt="image" src="https://github.com/user-attachments/assets/de7df0f3-478b-4507-8722-b9550820caa2">
```

Comment:
let's start having localized READMEs on each part of the project, similar to what we did in Nebula. What do you think? So this block could be under `app/integrations/discord/README.md` and if you'd like we can reference it here on the main README

Reply:
look! https://github.com/bro-garden/privy/blob/80024c2e792217547e4d037e7d2003d379dd9dae/README.md